manageiq-operator/api/v1alpha1/helpers/miq-components/httpd_conf.go

```diff
@@ -39,35 +39,35 @@ Options SymLinksIfOwnerMatch
   RequestHeader set Host %[1]s
   RequestHeader set X-Forwarded-Host %[1]s
 
+  # Send API requests to the API pods
+  ProxyPass /api %[4]s://web-service:3000/api
```

Comment:
Should the numbered placeholders be reordered?

Reply:
Done